cmake/install-rules.cmake

```diff
@@ -5,7 +5,7 @@ include(CMakePackageConfigHelpers)
 include(GNUInstallDirs)
 
 if(BUILD_PRINTER)
-  install(TARGETS ifc-printer DESTINATION . COMPONENT Printer)
+  install(TARGETS ifc-printer DESTINATION bin COMPONENT Printer)
```

Comment:
Do not hardcode destinations. They must always use [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) provided variables (`CMAKE_INSTALL_BINDIR` in this case), or be omitted as the correct defaults are used since CMake 3.14 (values taken from the module if included prior).